pkg/server/backend_manager.go

```diff
@@ -85,7 +88,24 @@ type backend struct {
 func (b *backend) Send(p *client.Packet) error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
-	return b.conn.Send(p)
+	// https://github.com/grpc/grpc-go/issues/1229
+	// wrap a timer for around SendMsg to avoid blocking grpc call
+	// (e.g. stream is full)
+	errChan := make(chan error, 1)
+	go func() {
```

Comment:
Is it possible for this goroutine to start leaking if b.conn.Send can block forever?

Reply:
My thinking is no, because when the stream closes eventually from returning from Send, this will return an io.EOF but I'm not 100% confident about it.

Reply:
Although I kinda agree that this change just masks the real problem. Thanks!

Reply:
Yes. 
I mention the buffered channel only because that is needed to be able free up the goroutine and the channel. Thanks'